internal/subshell/bash/bash.go

```diff
@@ -63,6 +63,10 @@ func (v *SubShell) WriteUserEnv(cfg sscommon.Configurable, env map[string]string
 		return errs.Wrap(err, "RcFile failure")
 	}
 
+	if _, pathExists := env["PATH"]; pathExists && runtime.GOOS == "windows" {
+		env["PATH"] = filepath.ToSlash(env["PATH"])
+	}
```

Comment:
Couple of issues:

- This isn't bashifying the path, it's just replacing the path separator characters. I can see you the actual bashify logic is in rcfile.go, so what is this trying to do?
- Comment explaining what it does would be helpful here
- You are calling `filepath.ToSlash()` on a value that is itself not a path, but contains a variety of paths. Especially on windows this could create some weird bugs.

I imagine we will have to unpack this and bashify each individual path. If this is the case keep in mind that `osutils.BashifyPath()` calls out to `bash -c`. So calling it individually for each path would make this very slow. We might want to implement a variation of this that can bashify multiple paths with one call.

Reply:
This only happens when installing the State Tool from Git Bash on Windows, or when running state deploy. The incoming PATH contains only State Tool paths, so we are not modifying any existing Windows PATH entries. In my experiments with the installer, swapping '\' for '/' was enough, but this is not the case for deploy, so I will go back to splitting and bashifying each path component (which has only ever been 1 or 2 in my experiments).

Reply:
> In my experiments with the installer, swapping '\' for '/' was enough

This is not an assumption we can make. There will for certain be situations where this is not true.

Reply:
Okay, each path component is now bashified.